test/bdd/steps/datalayer.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable no-unused-expressions, max-len, no-await-in-loop */
+
+const {
+    Then,
+} = require('cucumber');
+const { expect } = require('chai');
+
+const httpApiHelper = require('./lib/http-api-helper');
+const utilities = require('./lib/utilities');
+const Utilities = require('../../../modules/Utilities');
```

Comment:
Avoid code from outside tests.

Reply:
ok, from Utilities am using only isZeroHash(), which I can also define in ./lib.utilities, will fix